cosmic_ray/plugins.py

```diff
@@ -41,8 +41,11 @@ def get_operator(name):
 
 
 def operator_names():
-    """Get an iterable of all operator names."""
-    return (
+    """Get all operator names.
+
+    Returns: A sequence of operator names.
+    """
+    return tuple(
```

Comment:
Is your PR based on something other than `master`? This change (and some of your others) looks to be identical to one we made on the `fix-init` branch.